File: src/AdventureWorks.Application/DataEngine/Production/ProductModelProductDescriptionCulture/Commands/Queries/GetProductModelProductDescriptionCultures/ProductModelProductDescriptionCultureLookupModel.cs
using System;
using AutoMapper;
using AdventureWorks.Application.Interfaces.Mapping;
using AdventureWorks.BaseDomain.Entities.Production;
using System.Collections.Generic;
using AdventureWorks.Common.Extensions;
using AdventureWorks.BaseDomain.Enums;
using AdventureWorks.BaseDomain.Interfaces;
using AdventureWorks.BaseDomain.CustomTypes;

namespace AdventureWorks.Application.DataEngine.Production.ProductModelProductDescriptionCulture.Queries.GetProductModelProductDescriptionCultures
{
    public partial class ProductModelProductDescriptionCultureLookupModel : IHaveCustomMapping
    {
        public int ProductModelID { get; set; }
        public int ProductDescriptionID { get; set; }
        public string CultureID { get; set; }
        public DateTime ModifiedDate { get; set; }

        #region Navigation Properties

        // Uncomment the following line if you need child navigation properties (can cause a poor performance)
        /*  */
        // Uncomment the following line if you need parent navigation properties
        /*  */
        #endregion

        public void CreateMappings(Profile configuration)
        {
            configuration.CreateMap<Domain.Entities.Production.ProductModelProductDescriptionCulture, ProductModelProductDescriptionCultureLookupModel>().IgnoreMissingDestinationMembers().CompensateWithDestinationValues();
            configuration.CreateMap<ProductModelProductDescriptionCultureLookupModel, BaseProductModelProductDescriptionCulture>().IgnoreMissingDestinationMembers();
        }
    }
}

File: src/AdventureWorks.Web.Api/ClientApp/src/app/models/data/entities/production/product-model-product-description-culture/product-model-product-description-culture-lookup-model.ts

export interface IProductModelProductDescriptionCultureLookupModel {
  productModelID: number;
  productDescriptionID: number;
  cultureID: string;
  modifiedDate: Date | string;

  // Uncomment the following line if you need child navigation properties (can cause a poor performance)
  /*  */
  // Uncomment the following line if you need parent navigation properties
  /*  */
}

export class ProductModelProductDescriptionCultureLookupModel implements IProductModelProductDescriptionCultureLookupModel {
  productModelID: number;
  productDescriptionID: number;
  cultureID: string;
  modifiedDate: Date | string;

  // Uncomment the following line if you need child navigation properties (can cause a poor performance)
  /*  */
  // Uncomment the following line if you need parent navigation properties
  /*  */

  constructor(init?: Partial<ProductModelProductDescriptionCultureLookupModel>) {
    if (!!init) {
      Object.assign<ProductModelProductDescriptionCultureLookupModel, Partial<ProductModelProductDescriptionCultureLookupModel>>(this, init);
    }
  }
}
